app/assets/stylesheets/admin/views/_document-collection-group-memberships-index.scss

```diff
@@ -10,3 +10,14 @@
     white-space: nowrap;
   }
 }
+
+.app-view-document-collection-group-memberships-index__heading-links {
+  display: flex;
+  align-items: center;
+
+  & > p:first-child {
+    margin-right: 10px;
```

Comment:
Can we use Design System [Responsive spacing](https://design-system.service.gov.uk/styles/spacing/#responsive-spacing) here rather than explicit values?

Reply:
Done!

Reply:
please remove 
margin-right: 10px; padding-right: 10px;
As you have already added responsive spacing for these